packages/manager/modules/beta-preference/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@ovh-ux/manager-beta-preference",
+  "version": "0.0.0",
```

Could be directly settled as:
```suggestion
  "version": "1.0.0",
```